### DISCUSSION TITLE
Discussion: "# issue"

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [SKbarbon/Flet_StoryBoard#8] [General]
Let's talk here about each of the # issues I mentioned.

**Comment by ndonkoHenri** [comment]
https://github.com/SKbarbon/Flet_StoryBoard/blob/de553f4223d389d1bd71d82caaaa5ff8ea558d05/Flet_StoryBoard/widgets/widgets/open_url.py#L93

`Row` "alignment" can't have value 'right'. See allowed properties [here](https://flet.dev/docs/controls/row#alignment) please.

**Reply by SKbarbon** [reply]
But it works fine!

**Comment by ndonkoHenri** [comment]
https://github.com/SKbarbon/Flet_StoryBoard/blob/de553f4223d389d1bd71d82caaaa5ff8ea558d05/Flet_StoryBoard/engines/suggesting_engine.py#L16

and

https://github.com/SKbarbon/Flet_StoryBoard/blob/de553f4223d389d1bd71d82caaaa5ff8ea558d05/Flet_StoryBoard/widgets/widgets/row.py#L97

`Column` "scroll" property can't have value `True`. See allowed properties [here](https://flet.dev/docs/controls/column#scroll) please.

**Reply by SKbarbon** [reply]
It works fine as expected!

**Reply by SKbarbon** [reply]
@ndonkoHenri

**Comment by ndonkoHenri** [comment]
https://github.com/SKbarbon/Flet_StoryBoard/blob/de553f4223d389d1bd71d82caaaa5ff8ea558d05/Flet_StoryBoard/engines/suggesting_engine.py#L159

You appended several controls to the column, so I thought an update() could be useful. If you don't think it's useful in this case, no prob. Simply remove my comment line from the code.

**Comment by ndonkoHenri** [comment]
Let me know what you think of the stuffs I pointed out. 
If you correct the mistakes, make sure to delete the corresponding comment lines please.
